### PR TITLE
API 6.6; remove Typegram proxy in favour of ApiMethods, Ret, Opt

### PR DESCRIPTION
**Pull request by MKRhere**
This will need a Typegram major version; I've edited README to reflect new changes. Make sure the changes are sound :)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -37,42 +37,14 @@ In fact, this pattern is used for various types, namely:
...
-There is no further structure applied to this.
-
-Each method takes just a single argument with a structure that corresponds to the object expected by Telegram.
-If you need to directly access that type, consider using `Opts<M>` where `M` is the method name (e.g. `Opts<'getMe'>`).
-
-Note that `Opts<M>` will give you an empty object type (i.e. `{}`) for methods that do not take any parameters.
-That is to say, it will not give you a type error or `undefined` (as opposed to something like `Parameters<Telegram['getMe']>[0]`).
-
-Each method returns the object that is specified by Telegram.
-If you directly need to access the return type of a method, consider using `Ret<M>` where `M` is the method name (e.g. `Opts<'getMe'>`).
```
Comment (KnorpelSenf) [inline README.md:52]: Why did you remove this section? The methods are still there, no?

File: README.md [hunk 2]
```diff
@@ -87,9 +59,8 @@ As an example, you may want to be able to make calls to `sendDocument` with an o
 (Your code is then assumed to able to translate calls to `sendDocument` and the like to multipart/form-data uploads when supplied with an object alike `{ path: '/tmp/file.txt' }` in the `document` property of the argument object.)
 
 `typegram` cannot possibly know what objects you want to support as `InputFile`s.
-Consequently, the exposed type `InputFile` is merely an alias for `never`.
 
-However, you can specify your own version of what an `InputFile` is, hence effectively creating a completely new version of `typegram` with your custom `InputFile` type used throughout all affected methods and interfaces.
+However, you can specify your own version of what an `InputFile` is throughout all affected methods and interfaces.
 This is possible by what we call a _proxy type_.
```
Comment (KnorpelSenf) [inline README.md]: Proxy types are no longer used. It now is just a regular type param.

File: README.md (hunk 1 above)
Comment (MKRhere) [reply README.md:52]: I removed it because `Telegram` no longer exists; `Opts` needs `<F>`. This is confusing before you read about "Customizing `InputFile` and Accessing API Methods" which covers this.

Comment (KnorpelSenf) [reply README.md:52]: Alright.

File: README.md (hunk 2 above)
Comment (MKRhere) [reply README.md]: Removed reference to proxy type.

**Review by KnorpelSenf — APPROVED** [review]

File: README.md (hunk 2 above)
Comment (KnorpelSenf) [reply README.md]: Thanks
